apps/web/src/styles/outfit.css

```
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Outfit:wght@100..900&display=swap');
```

Stylelint (17.2.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Outfit:wght@100..900&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Outfit:wght@100..900&display=swap'" (import-notation)

(import-notation)